src/widgets/trainers/ui/trainers.scss

```diff
@@ -1,88 +1,10 @@
-.nodejs-trainer {
-  & .content {
+.trainers {
```

Comment:
seems like you forgot to convert this one to modules also

Reply:
I suppose, it's out of scope - we'll change it as part of the widget code refactoring.

Reply:
It's mentioned in issue, pt.4

Reply:
I asked Bogdan and we decided to remove this part of pt4 as it out of scope